docs/src/content/docs/dev/hmr-integration.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: hmrIntegration
+---
+
+Until now, we recommended `watchIntegration` to enable HMR in development. However:
```

Comment:
Is it even worth having this first paragraph?

I feel this should be a paragraph just in the update guide, but this page should just refer to `hmrIntegration` and the _new_ way of doing things.

People who have never used either don't need to know the history

Reply:
I think it's fine to keep it here for a few releases, then we'll just move it to the upgrade guide